src/main/java/propertyFiles/Bundle.properties

```diff
@@ -155,6 +155,7 @@ contact.support=Support
 contact.from=From
 contact.from.required=User email is required.
 contact.from.invalid=Email is invalid.
+contact.from.emailPlaceholder=valid@email.org
```

Comment:
@jp-tosca for consistency should we use name@email.xyz?

src/main/java/propertyFiles/citation.properties
175:datasetfieldtype.datasetContactEmail.watermark=name@email.xyz

<img width="1006" alt="Screenshot 2024-01-05 at 11 30 33 AM" src="https://github.com/IQSS/dataverse/assets/21006/d8b10c16-18c7-42c6-bf86-770012d0b874">

Reply:
@jp-tosca and I talked about this. I'll make the emails match.

Reply:
Added in dbab6ca.